tests/scattering1d/test_torch_scattering1d.py

```diff
@@ -122,11 +122,12 @@ def test_computation_Ux(backend, device, random_state=42):
     rng = np.random.RandomState(random_state)
     J = 6
     Q = 8
-    T = 2**12
-    scattering = Scattering1D(J, T, Q, average=False,
-                              max_order=1, out_type="dict", frontend='torch', backend=backend).to(device)
+    shape = 2**J
+    scattering = Scattering1D(J, shape, Q, average=False,
+                              max_order=1, out_type="dict", frontend='torch', 
+                              backend=backend).to(device)
```

Comment:
Why this change?

Reply:
i approve this. `T` wasn't a good notation here. It's really the shape (length of signal) we're passing as second argument, not the `T` (amount of averaging).